minidump/src/minidump.rs

```diff
@@ -666,17 +662,17 @@ fn read_codeview(
     location: &md::MINIDUMP_LOCATION_DESCRIPTOR,
     data: &[u8],
     endian: scroll::Endian,
-) -> Result<CodeView, failure::Error> {
-    let bytes = location_slice(data, location)?;
+) -> Result<CodeView, ()> {
```

Comment:
It might be more straightforward to make this return `Option<CodeView>` instead of using `()` as the error type. You'd be trading all the `.or(Err(()))?` for `.ok()?`.